README.md

```diff
@@ -12,6 +12,7 @@ The following images are currently available:
 
 - `trusty` - Legacy build image for older sites; running Ubuntu 14.04 and [this software](https://github.com/netlify/build-image/blob/trusty/included_software.md)
 - `xenial` - Default build image for all new sites; running Ubuntu 16.04 and [this software](https://github.com/netlify/build-image/blob/xenial/included_software.md)
+- `focal` - Opt-in beta build image; running Ubuntu 20.04 and [this software](https://github.com/netlify/build-image/blob/xenial/included_software.md)
```

Comment:
Until we actually have this available for selection in the UI, I wonder if we change this line to something like this?
```suggestion
- `focal` - In development and available for local testing; running Ubuntu 20.04 and [this software](https://github.com/netlify/build-image/blob/xenial/included_software.md)
```
I say "local testing" to refer to running the steps in the readme with the `focal` branch named in the `docker pull` command.

When we have it available in the UI, we can change to the "opt-in beta" language.